### PR TITLE
Use new extractor that uses the JSON description.

### PR DESCRIPTION
**Pull request by rockneurotiko**
A while ago I created this project that parses the Telegram bot API documentation and provides a JSON file with all standarized: https://github.com/rockneurotiko/telegram_api_json

I updated the project and improved it, and changed the extractor in order to use this project.

This actually fixed some problems (models without optional, and some wrong types).

In the future there idea is to use the description provided in order to generate documentation.

Also, I'm thinking about doing this directly on elixir when compiling, the problem would be that it would remove the `model` and `method` on `ex_gram.ex`, which actually are really useful in order to see the existing methods available and the diffs on releases, so probably we'll keep this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -324,7 +324,9 @@ Note: Only one of `token` and `bot` must be used.
 
 ### How it's made?
 
-There is a python script called `extractor.py`, that scrapes the Telegram Bot API documentation and prints to the stdout the lines needed to create all the methods and models, this auto generated lines uses two macros defined on `lib/ex_gram/macros.ex`: `method` and `model`.
+There is a python script called `extractor.py`, it uses the (`telegram_api_json`)[https://github.com/rockneurotiko/telegram_api_json] project that scrapes the Telegram Bot API documentation and provides a JSON with all the information, check the project description if you want to create your own projects that uses an standarized file to auto-generate the API.
```
Comment (Ironjanowar) [inline README.md:327]: The URL will be displayed here:
image

I suggest you remove the inverted commas from `telegram_api_json`:
```
... (telegram_api_json)[https://github.com/rockneurotiko/telegram_api_json] ...
```

Comment (rockneurotiko) [inline README.md:327]:
```suggestion
There is a python script called `extractor.py`, it uses the (telegram_api_json)[https://github.com/rockneurotiko/telegram_api_json] project that scrapes the Telegram Bot API documentation and provides a JSON with all the information, check the project description if you want to create your own projects that uses an standarized file to auto-generate the API.
```

File: README.md [hunk 2]
```diff
@@ -324,7 +324,9 @@ Note: Only one of `token` and `bot` must be used.
 
 ### How it's made?
 
-There is a python script called `extractor.py`, that scrapes the Telegram Bot API documentation and prints to the stdout the lines needed to create all the methods and models, this auto generated lines uses two macros defined on `lib/ex_gram/macros.ex`: `method` and `model`.
+There is a python script called `extractor.py`, it uses the (telegram_api_json)[https://github.com/rockneurotiko/telegram_api_json] project that scrapes the Telegram Bot API documentation and provides a JSON with all the information, check the project description if you want to create your own projects that uses an standarized file to auto-generate the API.
```
Comment (rockneurotiko) [inline README.md:327]:
```suggestion
There is a python script called `extractor.py`, it uses the [`telegram_api_json`](https://github.com/rockneurotiko/telegram_api_json) project that scrapes the Telegram Bot API documentation and provides a JSON with all the information, check the project description if you want to create your own projects that uses an standarized file to auto-generate the API.
```

File: README.md (hunk 1 above)
Comment (rockneurotiko) [reply README.md]: Thanks! The problem was the order of brackets, it's `[]()` and not `()[]` :joy:

**Pull request merged**